build/build-image/cross/Dockerfile

```diff
@@ -66,14 +82,11 @@ RUN mkdir $TMPDIR \
   && chmod a+rwx $TMPDIR \
   && chmod o+t $TMPDIR
 
-# Get the code coverage tool and goimports
-RUN go get golang.org/x/tools/cmd/cover \
-           golang.org/x/tools/cmd/goimports \
-    && go clean -cache
-
 # Download and symlink etcd. We need this for our integration tests.
 RUN export ETCD_VERSION=v3.2.24; \
   mkdir -p /usr/local/src/etcd \
   && cd /usr/local/src/etcd \
-  && curl -fsSL https://github.com/coreos/etcd/releases/download/${ETCD_VERSION}/etcd-${ETCD_VERSION}-linux-amd64.tar.gz | tar -xz \
+  && [ "$(uname -m)" = "x86_64" ] && (curl -fsSL https://github.com/coreos/etcd/releases/download/${ETCD_VERSION}/etcd-${ETCD_VERSION}-linux-amd64.tar.gz | tar -xz) || true \
+  && [ "$(uname -m)" = "aarch64" ] && (curl -fsSL https://github.com/coreos/etcd/releases/download/${ETCD_VERSION}/etcd-${ETCD_VERSION}-linux-arm64.tar.gz | tar -xz) || true \
+  && [ "$(uname -m)" = "ppc64le" ] && (curl -fsSL https://github.com/coreos/etcd/releases/download/${ETCD_VERSION}/etcd-${ETCD_VERSION}-linux-ppc64le.tar.gz | tar -xz) || true \
   && ln -s ../src/etcd/etcd-${ETCD_VERSION}-linux-amd64/etcd /usr/local/bin/
```

Comment:
need to change amd64 to the specific arch folder.

Reply:
Ack doing that now